src/nominatim_db/clicmd/api.py

```diff
@@ -213,6 +213,9 @@ def run(self, args: NominatimArgs) -> int:
         except napi.UsageError as ex:
             raise UsageError(ex) from ex
 
+        locales = _get_locales(args, api.config.DEFAULT_LANGUAGE)
+        napi.Formatter().localize_results(results, locales)
```

Comment:
Remove the old locale code in line 199.

Also: there are lots more places to change in this file. Search for `'ĺocales'`.